api/routes.py

```diff
@@ -394,6 +394,11 @@ def annotations_for_work(identifier_type, identifier):
 def borrow(identifier_type, identifier, mechanism_id=None):
     return app.manager.loans.borrow(identifier_type, identifier, mechanism_id)
 
+@library_route('/works/<license_pool_id>/fulfill/<mechanism_id>/<part>/rbdproxy/<bearer>')
+@has_library
+def proxy_rbdigital_patron_requests(license_pool_id, mechanism_id, part, bearer):
+    return app.manager.rbdproxy.proxy(bearer)
+
```

Comment:
Since license_pool_id, mechanism_id, and part are not used, can we remove them from the URL?

Reply:
We could. I left them in so that the URLs would look similar to the fulfillment URLs in any logs, but we don't need them at all to achieve the functionality.

Reply:
Since you're the main audience for this branch I'm OK with whatever makes things easy for you.